src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -364,20 +364,25 @@
     <!-- Hook the target that fetches the CFamily files into to the build process. 
          This needs to be set after the SDK targets have been imported.
     -->
-    <PrepareForBuildDependsOn>$(PrepareForBuildDependsOn);GetCFamilyFiles</PrepareForBuildDependsOn>
+    <CoreCompileDependsOn>$(CoreCompileDependsOn);GetCFamilyFiles</CoreCompileDependsOn>
   </PropertyGroup>
 
-  <Target Name="GetCFamilyFiles">
-    <Message Importance="high" Text="Fetching the CFamily files..." />
+  <ItemGroup>
+      <!-- The exe is a build-time utility i.e. we need to reference it so it is built when we need it,
+           but we don't want it to be copied to the bin directory or included in the VSIX.
+           See the notes in the tool project file for more information.
+      -->
+    <ProjectReference Include="..\..\build\tools\CFamilyJarPreProcessor\CFamilyJarPreProcessor.csproj">
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <IncludeOutputGroupsInVSIX>GetItemsToIncludeInVSIX_NoOp</IncludeOutputGroupsInVSIX>
+    </ProjectReference>
+  </ItemGroup>
 
-    <!-- Note: building the helper project explicitly using MSBuild rather than referencing it via a ProjectReference.
-         This makes is easier for us to control when the build is performed, so we can trigger it early enough
-         to include the downloaded files as normal Content files in this project.
-         Also, we are using the "Exec" task to invoke MSBuild.exe rather the "MSBuild" task when building
-         DownloadSonarPlugin.csproj to avoid getting file-locking errors when rebuilding.
```

Comment:
Hopefully this new very will actually fix the file-locking issues.
It should, because we are now referencing the tool via a `ProjectReference`, so MSBuild will be able to correctly handle the build dependencies, and will only build the tool project once.